columnize_test.go

```diff
@@ -74,6 +79,42 @@ func TestColumnWidthCalculator(t *testing.T) {
 	}
 }
 
+func BenchmarkColumnWidthCalculator(b *testing.B) {
+	// Generate the input
+	input := []string{
+		"UUID A | UUID B | Column C | Column D",
+	}
+
+	format := "%s|%s|%s|%s"
+	short := "short"
+
+	uuid := func() string {
+		buf := make([]byte, 16)
+		if _, err := crand.Read(buf); err != nil {
+			panic(fmt.Errorf("failed to read random bytes: %v", err))
+		}
+
+		return fmt.Sprintf("%08x-%04x-%04x-%04x-%12x",
+			buf[0:4],
+			buf[4:6],
+			buf[6:8],
+			buf[8:10],
+			buf[10:16])
+	}
+
+	for i := 0; i < 1000; i++ {
+		l := fmt.Sprintf(format, uuid(), uuid(), short, short)
```

Comment:
It might uncover some more of the performance benefit if you sliced up the UUID's so they are not uniform in length. We are effectively skipping most of the format calculation it seems since the length of the UUID's and the "short" string are constant. Not required though, just nice to have.